packages/vitest/src/node/error.ts

```diff
@@ -76,6 +77,8 @@ export async function printError(error: unknown, ctx: Vitest, options: PrintErro
     + '\n- The error was thrown, while Vitest was running this test.'
     + '\n- This was the last recorded test before the error was thrown, if error originated after test finished its execution.'))
   }
+  if (afterEnvTeardown)
+    ctx.logger.error(c.red('This error was caught after test environment was torn down'))
```

Comment:
Can we add possible solutions here? Clear timeouts for example. Maybe it can also happen because of a running promise?

Reply:
How about:

```
This error was caught after test environment was torn down. Make sure to cancel any running tasks before test finishes:
- cancel timeouts using clearTimeout and clearInterval
- wait for promises to resolve using await keyword
```

Reply:
Sounds good 👍🏻